tests/gold_tests/body_factory/gold/http-get-304.gold

```diff
@@ -1,5 +1,4 @@
 HTTP/1.1 304 Not Modified
 Age: 0
 Connection: keep-alive
-Warning: 199 ApacheTrafficServer/8.0.0 Proxy received unexpected 304 response; content may be stale
-
+Warning: 199 ApacheTrafficServer/7.1.0 Proxy received unexpected 304 response; content may be stale
```

Comment:
I don't know, I'll have to check with Jason. I thought this would be an easy backport but it looks like it's not.